gp-templates/helper-functions.php

```diff
@@ -315,24 +386,162 @@ function references( $project, $entry ) {
 	 */
 	$show_references = apply_filters( 'gp_show_references', (bool) $entry->references, $project, $entry );
 
-	if ( ! $show_references ) return;
+	if ( ! $show_references ) {
+return;
+	}
 	?>
 	<dl><dt>
 	<?php _e( 'References:', 'glotpress' ); ?>
 	<ul class="refs">
-		<?php
-		foreach( $entry->references as $reference ):
+	<?php
+		foreach ( $entry->references as $reference ) :
 			list( $file, $line ) = array_pad( explode( ':', $reference ), 2, 0 );
-			if ( $source_url = $project->source_url( $file, $line ) ):
-				?>
-				<li><a target="_blank" tabindex="-1" href="<?php echo $source_url; ?>"><?php echo $file.':'.$line ?></a></li>
-				<?php
+			if ( $source_url = $project->source_url( $file, $line ) ) :
+	?>
+				<li><a target="_blank" tabindex="-1" href="<?php echo $source_url; ?>"><?php echo esc_html( $file ) . ':' . esc_html( $line ); ?></a></li>
+	<?php
 			else :
-				echo "<li>$file:$line</li>";
+				echo '<li>' . esc_html( $file ) . ':' . esc_html( $line ) . '</li>';
 			endif;
 		endforeach;
-		?>
+	?>
 	</ul></dt></dl>
+	<?php
+}
+
+/**
+ * Output the bulk actions toolbar in the translations page.
+ *
+ * @param string $bulk_action     The URL to submit the form to.
+ * @param string $can_write       Can the current user write translations to the database.
+ * @param string $translation_set The current translation set.
+ * @param string $location        The location of this toolbar, used to make id's unique for each instance on a page.
+ */
+function gp_translations_bulk_actions_toolbar( $bulk_action, $can_write, $translation_set, $location = 'top' ) {
```

Comment:
This gp_translations_bulk_actions_toolbar function has been duplicated twice so there's currently 3 instances which causes a fatal;
[19-Oct-2018 05:13:06 UTC] PHP Fatal error:  Cannot redeclare gp_translations_bulk_actions_toolbar() (previously declared in /app/public/wp-content/plugins/glotpress/gp-templates/helper-functions.php:420) in /app/public/wp-content/plugins/glotpress/gp-templates/helper-functions.php on line 546

Reply:
my fault on resolving last conflicts